packages/app/jest.config.js

```diff
@@ -2,6 +2,8 @@
 const tsPreset = require('ts-jest/jest-preset')
 const mongoPreset = require('@shelf/jest-mongodb/jest-preset')
 
+process.env.TZ = 'GMT' // ensure all tests use GMT time, both locally and on CI servers
```

Comment:
I tried so hard to make this work in `jest.setup.ts`, with all the other environment variables. 😆  Setting Jest to use a `globaConfig` file also didn't work (broke something with `@shelf/jest-mongodb`, I think). So all that is to say, I see why this is here.